src/bin/poudriere-bulk.8

```diff
@@ -111,6 +114,17 @@ arguments may be specified at once.
 .El
 .Sh OPTIONS
 .Bl -tag -width "-B name"
+.It Fl b Ar name
+Specify the
+.Ar name
+of the binary package branch to use to prefetch packages.
```

Comment:
`-b` doesn't make me think fetch at all. In bulk.sh it is `-b branch` which makes me think 'branch' rather than 'binary'; I'm saying that at a quick glance of the manpage I might think this is saying which branch to build from rather than anything to do with fetching. I don't really have a better idea.
We're getting to the point of needing to use things like `--fetch`. I had been thinking `--seed` but `-f -F -S` are taken already and I had a plan to flip `-S` into `-s` IIRC.

Like Brooks comment I think this inevitably ends in someone asking for `REPO_URL` to be configurable so this flag interface may need to forward think about that.